include/seccomp.h.in

```diff
@@ -69,6 +69,11 @@ enum scmp_filter_attr {
 	SCMP_FLTATR_API_TSKIP = 5,	/**< allow rules with a -1 syscall */
 	SCMP_FLTATR_CTL_LOG = 6,	/**< log not-allowed actions */
 	SCMP_FLTATR_CTL_SSB = 7,	/**< disable SSB mitigation */
+	SCMP_FLTATR_CTL_OPTIMIZE = 8,	/**< filter optimization level:
+					 * 0 - currently unused
+					 * 1 - default (uses syscall priorities)
```

Comment:
Describing the optimization methods in the comments is admittedly hard, but I think we can do better than "syscall priorities".  I would suggest "rules weighted by priority and complexity".